ee_tests/oc-get-project-logs.sh

```diff
@@ -61,4 +61,8 @@ then
   fi
 fi
 
+buildName=$(oc get builds -n $1 | awk '{print $1}' | tail -1)
+echo ---------- Get stack analysis URL for $buildName ---------------
```

Comment:
Please echo also the actual command that is executed (as are all other commands)